src/main/webapp/search-interview.js

```diff
@@ -18,42 +18,67 @@ function onSearchInterviewLoad() {
   loginInfo.then(getUserOrRedirectRegistration);
 }
 
-// Should query Datastore for appropriate interviews and render them on the
+// Queries Datastore for available interview times and renders them on the
 // page.
 function loadInterviews() {
-  // TODO: fetch and get all interviews instead of unhiding something hidden
   const searchResultsDiv = document.getElementById("search-results");
   searchResultsDiv.removeAttribute("hidden");
+
+  fetch(`/load-interviews?timeZoneOffset=${browserTimezoneOffset()}`)
+    .then(response => response.text())
+      .then(interviewTimes => {
+        interviewTimesDiv().innerHTML = interviewTimes;
+      });
+}
+
+function interviewTimesDiv() {
+  return document.getElementById('interview-times-container');
+}
+
+function browserTimezoneOffset() {
```

Comment:
This is duplicated from https://github.com/googleinterns/InterviewMe/blob/2d33d23e54a4f731c9f2c474ce1a6aacfbef9557/src/main/webapp/availability.js#L43-L46

Please deduplicate all duplicated code by moving it into a common file where possible.

Reply:
Done